safe-global/safe-react
# fix: throttle balance syncer + remove WC patch

## What it solves
Resolves `eth_getBalance` over-polling.

## How this PR fixes it
Onboard's balance state syncer has been throttled from every 1s to every 1h. As such, the patched WC module has also been reverted to the default.

## How to test it
- Connect to WC and observe no `eth_getBalance` over-polling.
- Connect to a Ledger/Trezor and observe the same.
- Pairing should work as expected.
